server_py/flatgov/templates/bills/detail.html

```diff
@@ -286,141 +301,184 @@ <h4>Understand the Context</h4>
         </div>
         </div>
     </ul>
-    <div class="tab-content light-gray-tabs p-4" id="crsReportContent">
+    <div class="tab-content light-gray-tabs p-4" id="myTabContent">
         <div class="tab-pane fade show active" id="crs-report" role="tabpanel" aria-labelledby="crs-report-tab">
             <h6 class="pt-3">
-                View CRS reports for {{ bill.get_type_abbrev }} {{ bill.number }} in the {{ bill.congress }}th Congress
+                View all CRS Reports for {{ bill.type_abbrev }} {{ bill.meta.number }} in the {{ bill.meta.congress }}th Congress
```

Comment:
this one has the full congress spelling, I'd make sure it's consistent

Reply:
Also, why is it sometimes bill.meta.number vs. bill.meta.congress? Should it all be the same or is there a difference